### DISCUSSION TITLE
Problem installing spanish model

### DISCUSSION THREAD

**Discussion by wvalverde67** [explosion/spaCy#7362] [Help: Installation] [answered]
## How to reproduce the problem

```bash
python3 -m spacy download es_core_new_sm
Result:
✘ No compatible package found for 'es_core_new_sm' (spaCy v3.0.1)

```

## Your Environment

* Operating System: Ubuntu 20.04
* Python Version Used: 3.8.5
* spaCy Version Used: 3.0.1
* Environment Information:

**Comment by svlandeg** [comment, marked as answer]
It looks like you have a small typo: it's `en_core_news_sm` - "news", not "new" ;-)
